dashboard/src/scenes/person/MedicalFile.js

```diff
@@ -747,11 +747,11 @@ export function MedicalFile({ person }) {
             const treatmentResponse = isNewTreatment
               ? await API.post({
                   path: '/treatment',
-                  body: prepareTreatmentForEncryption(values),
+                  body: prepareTreatmentForEncryption({ ...values, user: values.user || user._id }),
```

Comment:
pas besoin pour la création, c'est déjà géré

```suggestion
                  body: prepareTreatmentForEncryption(values),
```